### PR TITLE
[cluster_test] Add --emit-tx option

### PR DESCRIPTION
**Pull request by andll**
This commit adds option for cluster test to emit transactions, that can be used for load testing.

We submit transactions at fixed rate, number of workers and submission rate per worker is defined by user.
Each transaction is 'ring transfer' - every worker thread has its own set of account addresses and it transfers money from one account to another. Account addresses are pre-minted with high amount of libra, so that practically those accounts always have money to perform transfer.
There is still room for improvement here: (1) currently we may fill mempool if we run this test for infinitely long time. It also does not allow to properly measure finality time. (2) We also don't currently wait for transactions to be committed

We reuse bench library for minting, but load generation part is different.
Main differences between tx_emitter and bench

(1) Transactions are submitted at fixed rate, instead of batch submitting a lot of transactions at once
(2) Using generated keys instead of wallet library - wallet is single threaded, using generated keys allows to sign transactions in multiple threads

This is not yet integrated in experiment run, but instead submits transactions as separate --emit-tx

### CODE REVIEW

File: testsuite/cluster_test/src/tx_emitter.rs [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+use proto_conv::IntoProto;
+use std::{
+    env,
+    str::FromStr,
+    sync::Arc,
+    thread,
+    time::{Duration, Instant},
+};
+
+pub struct TxEmitter {
```
Comment (dmitri-perelman) [inline testsuite/cluster_test/src/tx_emitter.rs]: nit: could you move it down? (it's hidden between the `use` directives right now.

File: testsuite/cluster_test/src/tx_emitter.rs [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+
+    fn create_ac_clients(cluster: &Cluster) -> Vec<(Instance, AdmissionControlClient)> {
+        let mut clients = vec![];
+        let threads_per_client = get_env("THREADS_PER_CLIENT", THREADS_PER_CLIENT_DEFAULT);
+        for instance in cluster.instances() {
+            let address = format!("{}:8000", instance.ip());
+            for _ in 0..threads_per_client {
+                let env_builder = Arc::new(EnvBuilder::new().name_prefix("ac-grpc-").build());
+                let ch = ChannelBuilder::new(env_builder)
+                    .primary_user_agent(&format!("grpc/client-{}", instance.short_hash()))
```
Comment (dmitri-perelman) [inline testsuite/cluster_test/src/tx_emitter.rs]: I wonder whether we need it (remove may be?)

File: testsuite/cluster_test/src/tx_emitter.rs (hunk 1 above)
Comment (andll) [reply testsuite/cluster_test/src/tx_emitter.rs]: Oh lol, yeah, probably some mix of cargo fmt and me being sloppy

**Pull request closed without merging**